### PR TITLE
e2e tests for multiple versions of kube

### PR DESCRIPTION
**Pull request by lucasreed**
I realize now that by squashing down to a single commit I lost @kimschles contributions to this branch. Sorry about that and thanks for helping with this @kimschles !

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
-version: 2
+version: 2.1
+
+orbs:
+  rok8s-scripts: fairwinds/rok8s-scripts@dev:unprivileged_commands_orb
```
Comment (sudermanjr) [inline .circleci/config.yml]: I think this has been released in the `11` series now.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+
+executors:
+  golang12:
+    docker:
+      - image: circleci/golang:1.12-stretch
```
Comment (sudermanjr) [inline .circleci/config.yml]: Is there any reason not to go to 1.13?

File: .circleci/config.yml (hunk 1 above)
Comment (lucasreed) [reply .circleci/config.yml]: Good catch, I forgot to change that!

File: .circleci/config.yml (hunk 2 above)
Comment (lucasreed) [reply .circleci/config.yml]: Good point. The go.mod says go v1.13 so that makes sense.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,4 +1,12 @@
-version: 2
+version: 2.1
+
+orbs:
+  rok8s-scripts: fairwinds/rok8s-scripts@11
+
+executors:
+  golang12:
```
Comment (sudermanjr) [inline .circleci/config.yml]: Do you want to name this just `golang` since it's a newer version now?

Comment (lucasreed) [reply .circleci/config.yml]: Good catch, thanks!

**Pull request merged**